### PR TITLE
Remove OpenCL ICD

### PR DESCRIPTION
**Pull request by Rbiessy**
Working fine for both local and cross-compilation toolchain.

### CODE REVIEW

File: third_party/sycl/sycl/BUILD.tpl [hunk 1]
```diff
@@ -45,10 +45,6 @@ cc_library(
         sycl_library_path("ComputeCpp")
     ],
     includes = ["include/"],
-    linkstatic = 0,
```
Comment (Rbiessy) [inline third_party/sycl/sycl/BUILD.tpl:48]: I removed that since it is dynamic by default

**Review by DuncanMcBain — APPROVED** [review]

**Comment by DuncanMcBain** [comment]
Let me know if and when you merge this - I'll probably just delete our fork of the ICD loader since it will no longer be required.

**Review by lukeiwanski — APPROVED** [review]
Looks good!

**Comment by lukeiwanski** [comment]
@DuncanMcBain  merged!

**Pull request merged**